extensions/amp-story/0.1/amp-story.css

```diff
@@ -16,7 +16,6 @@
 
 @import './amp-story-desktop.css';
 @import './amp-story-user-overridable.css';
-@import './amp-story-share.css';
```

Comment:
Should this be removed from 0.1 when the changes are only made to 1.0?

Reply:
Omg no, thanks for spotting this!!!